### DISCUSSION TITLE
The password???

### DISCUSSION THREAD

**Discussion by daonhan213** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1230] [General]
"I've accidentally forgotten my password, what should I do now?"
 
<img width="349" alt="Screen Shot 2024-05-22 at 12 07 38 AM" src="https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/170453793/f5189dbe-77e5-4ae5-9fca-1b7d42fee1ab">

**Comment by MatthiasKerbl** [comment]
Hi @daonhan213,

Here is the official Apple Support page about what to do when you forget your Mac login password:
https://support.apple.com/en-us/102633

Best,
Matthias
